packages/public/inspection-capture-web/src/PhotoCapture/PhotoCaptureHUDAddDamagePreview/CloseupPreview/CloseupPreview.tsx

```diff
@@ -0,0 +1,41 @@
+import { PixelDimensions, Sight } from '@monkvision/types';
+import { useTranslation } from 'react-i18next';
+import { useSightLabel } from '@monkvision/common';
+import { labels } from '@monkvision/sights';
+import { styles } from './CloseupPreview.styles';
+import { CancelButton } from '../CancelButton';
+import { DamageCounter } from '../DamageCounter';
+import { useCloseupPreviewStyle } from './hook';
+import { AddDamagePreviewMode } from '../../hooks';
+
+export interface CloseupPreviewProps {
+  sight?: Sight | undefined;
+  onCancel: () => void;
+  streamDimensions?: PixelDimensions | null;
+}
+
+export function CloseupPreview({ sight, onCancel, streamDimensions }: CloseupPreviewProps) {
+  const { t } = useTranslation();
+  const { label } = useSightLabel({ labels });
+  const style = useCloseupPreviewStyle();
+
+  const sightLabel = sight && label(sight);
+  const aspectRatio = `${streamDimensions?.width}/${streamDimensions?.height}`;
```

Comment:
if `streamDimensions` is either `null` or `undefined` (which it can be), `aspectRatio` is equal to `NaN`. Even though, by looking at your component, everything will still work properly with `streamDimensions = NaN`, I suggest we avoid having wrong values like `NaN` hanging around in our code, it might break something in the future and it would be annoying to debug.

So I suggest we simply put a default value :

```typescript
const aspectRatio = streamDimensions ? `${streamDimensions?.width}/${streamDimensions?.height}` : 16/9;
```